### PR TITLE
All deliveries

### PR DESCRIPTION
**Pull request by MaximBenkert**
Ticket 2
[Link zur User-Story] (https://github.com/neuefische/hh-java-23-1-teamproject-driver/issues/2)

### CODE REVIEW

File: backend/src/test/java/de/neuefische/driver/backend/controllers/DeliveryIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class DeliveryIntegrationTest {
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    ObjectMapper objectMapper;
```
Comment (Flooooooooooorian) [inline backend/src/test/java/de/neuefische/driver/backend/controllers/DeliveryIntegrationTest.java]: Remove unused variables

File: frontend/src/App.css [hunk 2]
```diff
@@ -1,38 +1 @@
-.App {
```
Comment (Flooooooooooorian) [inline frontend/src/App.css:1]: I would remove empty files

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -1,26 +1,48 @@
...
-    </div>
-  );
+    const [deliveries, setDeliveries] = useState<DeliveryModel[]>([])
+
+    useEffect(() => {
+        loadDeliveries()
+    }, [])
+
+    function loadDeliveries() {
+        axios.get("api/deliveries")
```
Comment (Flooooooooooorian) [inline frontend/src/App.tsx]:
```suggestion
        axios.get("/api/deliveries")
```

File: frontend/src/components/Gallery.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+export default function Gallery(props: GalleryProps) {
+    return (
+        <div>
+            <Typography sx={{fontSize: "1.5rem", padding: "1rem"}} variant="h2" component="h2">
+                Deliveries
+            </Typography>
+            <ul className="card-container">
+                {props.deliveries.map((delivery) => {
+                    return (
+                        <Card key={delivery.id} delivery={delivery}/>)
```
Comment (Flooooooooooorian) [inline frontend/src/components/Gallery.tsx]: Maybe use a specific name

File: frontend/src/components/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    const theme = createTheme({
+        typography: {
+            h1: {
+                fontSize: '2rem',
+                color: '#e0e0e0',
+                textAlign: "center",
+                padding: "1rem",
+                marginBottom: "0.7rem",
+                backgroundColor: "#ff6d00",
+            }}})
```
Comment (Flooooooooooorian) [inline frontend/src/components/Header.tsx:12]: i would move the theme into the app

**Review by Flooooooooooorian — APPROVED** [review]
LGTM

File: backend/src/test/java/de/neuefische/driver/backend/controllers/DeliveryIntegrationTest.java (hunk 1 above)
Comment (devToJu) [reply backend/src/test/java/de/neuefische/driver/backend/controllers/DeliveryIntegrationTest.java]: done

File: frontend/src/App.css (hunk 2 above)
Comment (devToJu) [reply frontend/src/App.css:1]: done

File: frontend/src/App.tsx (hunk 3 above)
Comment (devToJu) [reply frontend/src/App.tsx]: done

File: frontend/src/components/Gallery.tsx (hunk 4 above)
Comment (devToJu) [reply frontend/src/components/Gallery.tsx]: done

File: frontend/src/components/Header.tsx (hunk 5 above)
Comment (devToJu) [reply frontend/src/components/Header.tsx:12]: we solve it with the next styling

**Pull request merged**